packages/stress/src/demo.ts

```diff
@@ -34,45 +35,45 @@ async function spamInfo(count: number) {
         riverRegistry.getOperationalNodeUrls(),
     )
     for (let i = 0; i < count; i++) {
-        logger.log(`iteration ${i}`)
+        logger.info(`iteration ${i}`)
```

Comment:
@miguel-nascimento we generally want to avoid using template strings in the log message and reserve all variation in the metadata object. so something like:

`logger.info({ spamInfoLoopIndex: i }, 'spamInfo loop iteration')`

Reply:
in case you're curious, the reason we want this is to make log querying and log-derived metrics as simple as possible. it allows us to search for a simple static string